tests/unit/copying_manager_test.py

```diff
@@ -1497,181 +1430,25 @@ def test_whole_response_is_logged_on_non_success(self):
                 )
             finally:
                 if controller:
-                    controller.stop()
-
-    def test__read_checkpoint_state(self):
```

Comment:
Can you just double check this case is covered somewhere. I suspect it is but just want to double check.

Reply:
Yes, there is a `test_checkpoints` test case.
https://github.com/scalyr/scalyr-agent-2/blob/255f25119ece2093afb8a0eb97aaee09355e3ffe/tests/unit/sharded_copying_manager_tests/copying_manager_test.py#L331